<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Arbazs-MBP.domain.name" failures="1" tests="7" name="com.GeneralMathModule.ScientificNotationComponent" time="88.841" errors="0" timestamp="2023-07-01T20:57:06 IST" skipped="0">
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testConvertNumFromScientificToRegularNotation" time="13.238"/>
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testMultiplyTwoNumWithTwoScientificNotation" time="12.177"/>
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testDivideTwoNumOneOfThemIsScientificNotation2" time="15.296"/>
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testDivideTwoNumBothInScientificNotation" time="11.834"/>
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="ConvertNumFromRegularToScientificNotation" time="13.582"/>
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testDivideTwoNumOneOfThemIsScientificNotation" time="10.492">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.GeneralMathModule.ScientificNotationComponent.testDivideTwoNumOneOfThemIsScientificNotation(ScientificNotationComponent.java:300)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testDivideTwoNumOneOfThemIsScientificNotation -->
  <system-out/>
  <testcase classname="com.GeneralMathModule.ScientificNotationComponent" name="testMultiplyTwoNumWithSingleScientificNotation" time="12.222"/>
  <system-out/>
</testsuite> <!-- com.GeneralMathModule.ScientificNotationComponent -->
